ui/app/styles/core/link.scss

```diff
@@ -0,0 +1,27 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+.link {
+  background: transparent;
+  border: 0;
+  color: $blue;
+  cursor: pointer;
+  display: inline;
+  font: inherit;
+  line-height: normal;
+  margin: 0;
+  padding: 0;
+  font-weight: $font-weight-semibold;
+  user-select: text;
+
+  &[disabled] {
+    opacity: 0.5;
+    cursor: default;
+  }
+}
+
```

Comment:
ideally we would replace all instance of link-plain with a helper, `is-no-underline`. But I am waiting to do this in a future PR if we determine it's worth it and I have the time.